libs/partners/chroma/tests/unit_tests/test_vectorstores.py

```diff
@@ -1,15 +1,65 @@
-from langchain_core.embeddings.fake import (
-    FakeEmbeddings,
-)
+from typing import List
+
+from langchain_core.documents import Document
+from langchain_core.embeddings.fake import FakeEmbeddings
 
 from langchain_chroma.vectorstores import Chroma
 
 
-def test_initialization() -> None:
-    """Test integration vectorstore initialization."""
-    texts = ["foo", "bar", "baz"]
-    Chroma.from_texts(
-        collection_name="test_collection",
-        texts=texts,
-        embedding=FakeEmbeddings(size=10),
-    )
+class TestChromaSearch:
```

Comment:
There's a standard test suite  in langchain for vectorstores.

Here's an example of it being used: https://github.com/langchain-ai/langchain-postgres/blob/main/tests/unit_tests/test_vectorstore_standard_tests.py

Do you think you'll be able to convert to using it?

Reply:
Hi, we looked into this example but we have some questions about it and would like to ask for clarification. 
1. Do you want us to create another test file for the standard test, or do you just want us to inherit the `ReadWriteTestSuite` for the `test_vectorstore.py`? 
2. Do we need to create test cases for `async`?

Reply:
Just inherit and define the vectorstore implementation fixture. it should yield an empty vectorstore every time to unit tests relying on state introduces by other unit tests

```

class TestSync(ReadWriteTestSuite):
    @pytest.fixture()
    def vectorstore(self) -> Generator[VectorStore, None, None]:  # type: ignore
        """Get an empty vectorstore for unit tests."""
        with get_vectorstore(embedding=self.get_embeddings()) as vstore:
            vstore.drop_tables()
            vstore.create_tables_if_not_exists()
            vstore.create_collection()
            yield vstore
``

Reply:
Hi @eyurtsev, we just added standard test suite and fixed corresponding missing functions in Chroma